qlib/workflow/expm.py

```diff
@@ -155,28 +173,11 @@ class MLflowExpManager(ExpManager):
 
     def __init__(self, uri, default_exp_name):
         super(MLflowExpManager, self).__init__(uri, default_exp_name)
-        self._total_exps = 0
-        # get all the exps
-        self.experiments = self.list_experiments()
-
-    def start_exp(self, experiment_name=None, uri=None):
-        # create experiment
-        experiment = self.create_exp(experiment_name, uri)
-        # set up active experiment
-        self.active_experiment = experiment
-        # start the experiment
-        self.active_experiment.start()
-        self._total_exps += 1  # update exp num
-
-        return self.active_experiment
-
-    def end_exp(self, status):
-        if self.active_experiment is not None:
-            self.active_experiment.end(status)
-            self.active_experiment = None
-            self._total_exps -= 1
+        self.client = mlflow.tracking.MlflowClient(tracking_uri=self.uri)
 
     def create_exp(self, experiment_name=None, uri=None):
```

Comment:
It  is very hard to understand why we need such a complicated  `create_exp` function